bus/client/objects.go

```diff
@@ -82,8 +82,12 @@ func (c *Client) ObjectsBySlabKey(ctx context.Context, bucket string, key object
 }
 
 // ObjectsStats returns information about the number of objects and their size.
-func (c *Client) ObjectsStats() (osr api.ObjectsStatsResponse, err error) {
-	err = c.c.GET("/stats/objects", &osr)
+func (c *Client) ObjectsStats(ctx context.Context, opts api.ObjectsStatsOpts) (osr api.ObjectsStatsResponse, err error) {
```

Comment:
strictly speaking this is a compat issue and requires a major version bump

Reply:
Only the client method changed but the API change is backwards compatible. We don't have compatibility guarantees on our Go code right now.